### DISCUSSION TITLE
スライドモードのフルスクリーン表示がフルスクリーンにならない

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#473] [要望・アイデア・不具合]
スライドモードのフルスクリーンボタンを押すと、このような表示になります。

<img width="1912" alt="スクリーンショット 2023-05-02 23 21 01" src="https://user-images.githubusercontent.com/20086673/235694853-d374165d-8a4e-428f-a9a9-61842135cc32.png">

https://qiita.com/Qiita/items/4ff5873776992f0511d6

## 環境

- MacBook Pro (MacBookPro18,3)
- macOS 13.2.1 (22D68)
- Google Chrome バージョン: 112.0.5615.137（Official Build） （arm64）

**Comment by munierujp** [comment]
`.slideMode`に`max-width: 745px;`が指定されているのが原因っぽいです。
ひとまず次のようなCSSを当てることで回避できました。

```css
.slideMode.fullscreen {
  max-width: 100%;
}
```

**Reply by tanaka-asami415** [reply]
@munierujp 
ご報告ありがとうございます🙏
ご不便をおかけして申し訳ありません。こちら修正いたします！🙇

**Reply by kyntk** [reply]
@munierujp 大変お待たせしました。
修正が完了いたしました 🙇
